crypto-ffi/bindings/js/package.json

```diff
@@ -50,11 +50,7 @@
     "access": "public"
   },
   "scripts": {
-    "clean": "rm -f src/corecrypto.js src/corecrypto.d.ts",
-    "build": "bun run clean && bun build --target browser --format esm --outfile src/corecrypto.js src/CoreCrypto.ts",
-    "postbuild": "bun x dts-bundle-generator --project tsconfig.json -o src/corecrypto.d.ts --no-check --export-referenced-types false src/CoreCrypto.ts",
-    "test": "bun x wdio run wdio.conf.ts --spec test/wdio/*.test.ts && bun test",
-    "bench": "bun x wdio run wdio.conf.ts --spec benches/**/*.bench.ts --log-level warn"
+    "clean": "rm -f src/corecrypto.js src/corecrypto.d.ts"
```

Comment:
I'm not sure what the intention was here, but it seems weird that we removed everything except `clean`.